packages/playground/css/main.js

```diff
@@ -68,3 +68,10 @@ if (import.meta.env.DEV) {
 // inlined
 import inlined from './inlined.css?inline'
 text('.inlined-code', inlined)
+
+const glob = import.meta.glob('./glob-import/*.css')
```

Comment:
```suggestion

// glob
const glob = import.meta.glob('./glob-import/*.css')
```

a small comment for consistency in the file :)